src/main/groovy/life/qbic/datamodel/dtos/imaging/parameters/srm/SRMAnalysisParameters.groovy

```diff
@@ -35,86 +38,103 @@ class SRMAnalysisParameters extends AnalysisParameters {
      *
      * The convolution kernell is a window (usually rectangular).
      * Its size is described by a positive integer such as y x y
+     *
+     * A sample value would be 1.3 px
      */
-    private final int filterSize
+    private final Float filterSize
 
     /**
      * It describes the limiting spatial frequency of the microscope
      * Based on this value the smallest object resolvable by the microscope can be determined
      *
-     * It is given in cycles/millimeter
+     * It is given in photons, e.g. 1.5 photons
      */
-    private final int cutOff
+    private final Float cutOff
 
     /**
      * ROI stands for region of interest.
      * This region is defined by a number of pixels. All pixels contained within the area of interest describe it.
      *
-     * An image that comprises an ROI of 2x5 pixels has a ROI size of 10
-     * todo is it 2d or 3d, assume 2d since we name it region of interest and not VOI (volume of interest)
+     * An image that comprises an ROI of 2x5 pixels has a ROI size of 10 pixels
+     *
      */
-    private final int roiSize
+    private final Float roiSize
 
     /**
      * Describes the size of the image pixels.
      *
      * The {@link PixelSize} is characterized by 5 elements:
-     * (time, channel, x, y, z)
+     * (time, channel, x, y, z) in nm
      *
      * @see PixelSize
      */
     private final PixelSize pixelSize
 
     /**
      * The conversion factor applied for the analysis
+     * e.g 0.43 e-/ADU
      * todo maybe rename it to conversionFactor? what conversion is meant specifically?
```

Comment:
Thanks for your research. I think it would be best to wait for the stakeholder to specify the variable since its name and meaning were introduced by him.

Reply:
Good Idea, I added it to the master document so it won't get lost.

Reply:
Good Idea, I added the description to the Master document